frontend/device/devicelistener.lua

```diff
@@ -306,6 +306,18 @@ function DeviceListener:onToggleNoFlashOnSecondChapterPage()
     _toggleSetting("no_refresh_on_second_chapter_page")
 end
 
+function DeviceListener:onSetFlashOnPagesWithImages(toggle)
+    if toggle == true then
+        _unsetSetting("refresh_on_pages_with_images")
+    else
+        G_reader_settings:makeFalse("refresh_on_pages_with_images")
+    end
+end
+
+function DeviceListener:onToggleFlashOnPagesWithImages()
+    G_reader_settings:flipNilOrTrue("refresh_on_pages_with_images")
```

Comment:
This looks unbalanced, some of your branches using G_reader_settings while others use _unsetSettings - while all other non-yours branches use the wrappers _unsetSetting et al.
Indeed, none of these wrappers provide wrappers for flipNilOrTrue or makeFalse - and they look a bit useless doing nothing else than G_readersetting:make/flip... that one could do directly at the call site.
So, it's for @NiLuJe , who added them, to say how to proceeed :) (my preference would be to get rid of these wrappers).

Reply:
I think refactoring the wrappers to encapsulate the repetitive if/then on `onSet*` functions and support both cases (default=false, default=true) might make sense.

Reply:
I've pushed a commit to do so. Happy to revert if @NiLuJe prefers to keep them.

Reply:
Looks fine to me ;).

(I don't recall the full history of those weird wrappers, but they might have been doing more stuff at one point in time, and just became on-liners when I tweaked the *Settings API...)

Reply:
> (I don't recall the full history of those weird wrappers, but they might have been doing more stuff at one point in time, and just became one-liners when I tweaked the *Settings API...)

Or it was simply someone allergic to the fairly heinous naming scheme of said APIs ;p.